include/sparrow/fixed_size_layout.hpp

```diff
@@ -81,6 +80,12 @@ namespace sparrow
         reference operator[](size_type i);
```

Comment:
As discussed directly with @Klaim, `fixed_size_layout` iterators aren't invalidated by read or write access via `operator[]`; though this might be the case of other layouts in the future, such as for #40.